Resources/Prototypes/Recipes/Crafting/improvised.yml

```diff
@@ -10,7 +10,34 @@
     sprite: Objects/Weapons/Melee/baseball_bat.rsi
     state: icon
   objectType: Item
+# Floof Section
+- type: constructionGraph
+  id: SleepingBag
+  start: start
+  graph:
+  - node: start
+    edges:
+    - to: sleepingbag
+      steps:
+      - material: Cloth
+        amount: 5
+        doAfter: 4
+  - node: sleepingbag
+    entity: SleepingBagFolded
 
+- type: construction
+  name: sleeping bag
+  id: sleeping_bag
+  graph: SleepingBag
+  startNode: start
+  targetNode: sleepingbag
+  category: construction-category-furniture
+  description: A sleeping bed, perfect for those calm nights under a stary sky.
+  icon:
+    sprite: Floof/Structures/sleepingbag.rsi
+    state: open
+  objectType: Item
```

Comment:
This would be better placed in /Prototypes/Floof/Recipes/Crafting/improvised.yml

Reply:
This is true